releasenotes/notes/dogstatsd-origin-detected-unified-98897086225b0783.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    dogstatsd: Implement new config option `origin_detection_unified`.
```

Comment:
I think the release note might not be clear enough for users. We need to specify the changes. Also, perhaps it should be an `upgrade` since we're introducing a change that can potentially add tags to the metrics